pandas/tests/series/test_operators.py

```diff
@@ -1398,45 +1399,73 @@ def test_ops_datetimelike_align(self):
 
     def test_operators_bitwise(self):
         # GH 9016: support bitwise op for integer types
-        index = list('bca')
+        s_0123 = Series(range(4), dtype='int64')
+        s_1111 = Series([1] * 4, dtype='int8')
 
-        s_tft = Series([True, False, True], index=index)
-        s_fff = Series([False, False, False], index=index)
-        s_tff = Series([True, False, False], index=index)
-        s_empty = Series([])
+        # We cannot wrap s111.astype(np.int32) with pd.Index because it
```

Comment:
can you exercise both lhs and rhs are scalars (with a Series/Index) (True, False) and also ``None/np.nan``

Reply:
i c you might have these cases below, if you do make add _scalars in the test name (and see if you can fully exercise them)